apps/emqx_management/src/emqx_mgmt_api_plugins.erl

```diff
@@ -428,6 +428,7 @@ update_boot_order(post, #{bindings := #{name := Name}, body := Body}) ->
 %% For RPC upload_install/2
 install_package(FileName, Bin) ->
     File = filename:join(emqx_plugins:install_dir(), FileName),
+    ok = filelib:ensure_dir(File),
```

Comment:
I don't think this would cover all situations like the users maybe use a device which not exist in all nodes, so I think an error catch & report also is necessary.

Reply:
This would mean that the plugins directory does not exist and cannot be created by EMQX.  Since that directory path should be writable by EMQX, I guess we should raise an error here. :thinking: 

I've fixed the Mix file so that the empty directory is added to the tarball, so this should be a no-op, unless it's a pathological case as I've described above.